python_modules/dagster/dagster/_config/field.py

```diff
@@ -305,7 +305,7 @@ def __init__(
                         "into of a config enum type {name}. You must pass in the underlying "
                         "string represention as the default value. One of {value_set}."
                     ).format(
-                        value_set=[ev.config_value for ev in self.config_type.enum_values],
+                        value_set=[ev.config_value for ev in self.config_type.enum_values],  # type: ignore
```

Comment:
unrelated?

Reply:
not unrelated, since `inst()` performs narrowing, we go from `Unknown/Any` to `ConfigType` above, resulting in the need for this `type: ignore` (since only a subclass of `ConfigType` have `enum_values`)